server/src/main/java/org/elasticsearch/index/mapper/MapperFeatures.java

```diff
@@ -46,7 +46,7 @@ public class MapperFeatures implements FeatureSpecification {
     static final NodeFeature IVF_NESTED_SUPPORT = new NodeFeature("mapper.ivf_nested_support");
     static final NodeFeature BBQ_DISK_SUPPORT = new NodeFeature("mapper.bbq_disk_support");
     static final NodeFeature SEARCH_LOAD_PER_SHARD = new NodeFeature("mapper.search_load_per_shard");
-    static final NodeFeature PATTERNED_TEXT = new NodeFeature("mapper.patterned_text");
+    static final NodeFeature PATTERN_TEXT = new NodeFeature("mapper.patterned_text");
```

Comment:
I assume we're intentionally not changing the node feature name for bwc reasons?